lnwallet/channel.go

```diff
@@ -1724,22 +1724,6 @@ func (lc *LightningChannel) restoreStateLogs(
 		return nil
 	}
 
-	// If we do have a dangling commitment for the remote party, then we'll
```

Comment:
> The
reason we can safely remove this is to observe that a pending remote
commit is always created from a commitDiff which only contains updates
made by _us_, and thus only taken from the localUpdateLog

I'm not sure that this is _always_ the case, as it's possible to have a situation wherein we commit in coming HTLCs for them for the first time (as a side effect of our commitment).

Reply:
Although, if we're committing these HTLCs to them for the first time, then it should already be in the remote log, as on start up we'll populate the remote log with incoming HTLCs on _our_ commitment.

Reply:
Yes, remote updates will be applied to our commit first, which will be used to populate the index. This commit should also not change behaviour.